Src/Plugins/Security/SEC1003.ReviewPotentiallySensitiveFiles.json

```diff
@@ -4,16 +4,111 @@
       "Name": "ReviewPotentiallySensitiveFiles",
       "Id": "SEC1003",
       "Level": "Error",
-      "FileNameAllowRegex": "(?i)\\.(pem|pkcs12|p12|pfx|asc|ovpn|rdp|mdf|sdf|sqlite|sqlite3|bek|tpm|fve|jks|psafe3|agilekeychain|kwallet|tblk|ppk)$",
+      "FileNameAllowRegex": "(?i)\\.(agilekeychain|asc|bek|fve|jks|kwallet|mdf|ovpn|p12|pem|pfx|pkcs12|ppk|psafe3|rdp|sdf|sqlite|sqlite3|tblk|tpm)$",
       "Description": "Review potentially sensitive files in versioned engineering content and other locations for security risk.",
       "Message": "'{0:scanTarget}' is {1:fileKind} that may compromise security if visible to attackers.",
-
       "MatchExpressions": [
+        {
+          "SubId": "1PasswordFile",
+          "FileNameAllowRegex": "(?i)\\.agilekeychain$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a 1Password manager database file" }
+        },
         {
           "SubId": "ASCIIArmoredFile",
-          "NameAllowList": "(?i)\\.asc$",
+          "FileNameAllowRegex": "(?i)\\.asc$",
           "ContentsRegex": ".",
           "MessageArguments": { "fileKind": "an ASCII armored file" }
+        },
+        {
+          "SubId": "MicrosoftBitLockerRecoveryKeyFile",
+          "FileNameAllowRegex": "(?i)\\.bek$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a Microsoft BitLocker recovery key file" }
+        },
+        {
+          "SubId": "WindowsBitlockerDataFile",
+          "FileNameAllowRegex": "(?i)\\.fve$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a Windows BitLocker full volume encrypted data file" }
+        },
+        {
+          "SubId": "JavaKeystoreFile",
+          "FileNameAllowRegex": "(?i)\\.jks$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a Java keystore file" }
+        },
+        {
+          "SubId": "KDEWalletManagerDatabaseFile",
+          "FileNameAllowRegex": "(?i)\\.kwallet$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a KDE Wallet Manager database file" }
+        },
+        {
+          "SubId": "MicrosoftSQLDatabaseFile",
+          "FileNameAllowRegex": "(?i)\\.mdf$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a Microsoft SQL database file" }
+        },
+        {
+          "SubId": "OpenVPNClientConfigurationFile",
+          "FileNameAllowRegex": "(?i)\\.ovpn$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "an OpenVPN client configuration file" }
+        },
+        {
+          "SubId": "P12CryptographicKeyFile",
+          "FileNameAllowRegex": "(?i)\\.p12$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a cryptographic key bundle" }
```

Comment:
>cryptographic [](start = 47, length = 13)

say 'a PKCS#12 cryptographic key bundle? This is the same description for PFX files, maybe we merge the rules? Add PKCS12 as well? I think these are all files driven by the same container spec, PKCS#12